### PR TITLE
feat(Core): update to laravel 9;

### PR DESCRIPTION
**Pull request by taras-pinchuk-uinno**

### CODE REVIEW

File: Abstracts/Tests/PhpUnit/TestCase.php [hunk 1]
```diff
@@ -55,11 +57,8 @@ protected function tearDown(): void
      */
     protected function refreshInMemoryDatabase(): void
     {
-        // Migrate the database
-        $this->migrateDatabase();
-
-        // Seed the database
-        $this->seed();
+        // Migrate the database and seed the database
+        $this->artisan('migrate', $this->migrateUsing());
```
Comment (taras-pinchuk-uinno) [inline Abstracts/Tests/PhpUnit/TestCase.php:61]: Will the seeding be performed here?

File: Generator/Stubs/model.stub [hunk 2]
```diff
@@ -4,9 +4,23 @@ declare(strict_types=1);
 
 namespace App\Containers\{{section-name}}\{{container-name}}\Models;
 
-use App\Ship\Parents\Models\Model;
-
-class {{class-name}} extends Model
+use App\Ship\Parents\Models\Model as ParentModel;
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+use OpenApi\Attributes\Tag;
```
Comment (taras-pinchuk-uinno) [inline Generator/Stubs/model.stub]: It seems to me that we do not need to add to the stubs library which is not in the dependencies

File: Abstracts/Tests/PhpUnit/TestCase.php (hunk 1 above)
Comment (dmitryvovk) [reply Abstracts/Tests/PhpUnit/TestCase.php:61]: Yep, by method migrateUsing

File: Generator/Stubs/model.stub (hunk 2 above)
Comment (dmitryvovk) [reply Generator/Stubs/model.stub]: OK.

**Pull request merged**